Files/ViewModels/ItemViewModel.cs

```diff
@@ -41,7 +41,7 @@ public class ItemViewModel : INotifyPropertyChanged, IDisposable
         private readonly SemaphoreSlim enumFolderSemaphore = new SemaphoreSlim(1, 1);
         private readonly SemaphoreSlim loadExtendedPropsSemaphore = new SemaphoreSlim(Environment.ProcessorCount, Environment.ProcessorCount);
         private readonly ConcurrentQueue<(uint Action, string FileName)> operationQueue = new ConcurrentQueue<(uint Action, string FileName)>();
-        private readonly SemaphoreSlim operationSemaphore = new SemaphoreSlim(1, 1);
+        private readonly ManualResetEventSlim operationEvent = new ManualResetEventSlim();
```

Comment:
Great work. Can you explain how ManualResetEventSlim would improve things here?

Reply:
It's not mandatory to switch from Semaphore  to ManualResetEvent. But in my mind Semaphore is for controlling access to a shared resource, while here we use it to signal that an item has been added to the queue (looks like an event job).

Also it simplifies code a little:
```cs
try {
operationSemaphore.Release();
} catch (Exception)
{
// Prevent semaphore handles exceeding
}
```
To:
```cs
operationEvent.Set();
```